README.md

```diff
@@ -127,11 +125,15 @@ seeks [discussions][join] and continuous improvement.
 [join]: https://join.chaostoolkit.org/
 
 From a code perspective, if you wish to contribute, you will need to run a
-Python 3.6+ environment. Please, fork this project, write unit tests to cover
+Python 3.6+ environment. Please, [fork][fork] this project, [sync][sync] it with the upstream repository, write unit tests to cover
```

Comment:
Unfortunately, those instructions *aren't* what we want, they're doing merge commits 😭